presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1007,12 +1007,16 @@ public ConnectorTableHandle createTemporaryTable(ConnectorSession session, List<
             List<String> partitionColumns = partitioning.getPartitionColumns();
             BucketFunctionType bucketFunctionType = partitioningHandle.getBucketFunctionType();
             switch (bucketFunctionType) {
+                case HIVE_CLUSTERING:
```

Comment:
The function is to create temp table. We should actually throw if we see a clustering when writing to temp tables.